bcipy/helpers/save.py

```diff
@@ -25,33 +24,36 @@ def save_json_data(data: dict, location: str, name: str) -> str:
     return str(path)
 
 
+def save_experiment_data(data, location, name) -> str:
+    return save_json_data(data, location, name)
+
+
+def save_field_data(data, location, name) -> str:
+    return save_json_data(data, location, name)
+
+
```

Comment:
It doesn't seem like these functions add much value. Maybe we just need to make the documentation for save_json_data more generic?

Reply:
I wanted a good abstraction, so if we changed how we loaded or validated the function wouldn't have to change.